stable_baselines3/common/noise.py

```diff
@@ -110,6 +113,7 @@ def __init__(self, base_noise: ActionNoise, n_envs: int):
 
         self.base_noise = base_noise
         self.noises = [copy.deepcopy(self.base_noise) for _ in range(n_envs)]
+        self._dtype = dtype
```

Comment:
I would drop dtype in `VectorizedActionNoise`, since the type is already specified in `base_noise`. It could lead to unintuitive thing, such as:

```python
from stable_baselines3.common.noise import VectorizedActionNoise, NormalActionNoise
import numpy as np

noise = NormalActionNoise(np.array([0.0, 0.0]), np.array([1.0, 1.0]), dtype=np.float64)
vec_noise = VectorizedActionNoise(noise, 4)
vec_noise().dtype # np.float32
```

Reply:
Thanks for the feedback!